attacker/attacker.go

```diff
@@ -95,7 +95,4 @@ func Attack(ctx context.Context, target string, resCh chan *Result, metricsCh ch
 		}
 	}
 	metrics.Close()
```

Comment:
To show metrics that cannot be measured until the attack is completed, we'd better send after closing metrics.
```suggestion
	metrics.Close()
	metricsCh <- newMetrics(&metrics)
```

Reply:
That actually makes a lot of sense. Will get to work on it